programs/pluto/kernel_xfrm_interface.c

```diff
@@ -246,7 +248,9 @@ static bool nl_newlink(const char *ipsec_if_name,
 					   "cannot find interface index for physical interface device %s", physical_if_name);
 				return false;
 			}
-			nl_addattr32(&req.n, sizeof(req.data), IFLA_XFRM_LINK, physical_if_index);
```

Comment:
If `physical_if_index` isn't being used, then is the `if_nametoindex(physical_if_name)` call needed?